build.gradle

```diff
@@ -4,7 +4,7 @@ import org.gradle.api.tasks.testing.logging.TestLogEvent
 
 plugins {
     id 'java'
-    id 'org.springframework.boot' version "2.7.6"
+    id 'org.springframework.boot' version "3.0.6"
```

Comment:
Is this upgrade required for the test to pass? Looks like this change was made when you hit that authentication issue but this didn't solve that